File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 5/PRAK605-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int main ()
{
    int matriks1[10][10],matriks2[10][10],hasil[10][10];
    int i,j,k,matrix,jumlah = 0;
    scanf("%d",&matrix);
    if (matrix > 0 && matrix <= 3){
    printf("Matriks A \n");
    for(i = 0; i < matrix; i++){
      for(j = 0; j < matrix; j++){
        scanf("%d", &matriks1[i][j]);
      }
    }
    printf("Matriks B \n");
    for(i = 0; i < matrix; i++){
      for(j = 0; j < matrix; j++){
        scanf("%d", &matriks2[i][j]);
      }
    }
    for(i = 0; i < matrix; i++){
      for(j = 0; j < matrix; j++){
        for(k = 0; k < matrix; k++){
          jumlah = jumlah + matriks1[i][k] * matriks2[k][j];
        }
        hasil[i][j] = jumlah;
        jumlah = 0;
      }
    }
    printf("Matriks AXB\n");
    for(i = 0; i < matrix; i++){
      for(j = 0; j < matrix; j++){
        printf("%d ", hasil[i][j]);
      }
      printf("\n");
    }
  }
}

File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 5/PRAK605-2210817210014-MuhammadQoharyRawidafany.py
matrix = int(input())
matriks1 = []
print('Matriks A')
for i in range (matrix):
    k = list(map(int, input().split()))
    matriks1.append(k)
matriks2 = []
print('Matriks B')
for i in range (matrix):
    k = list(map(int, input().split()))
    matriks2.append(k)
matrikskali = []
print('Matriks AXB')
for i in range (matrix):
    matrikskali.append([])
    for j in range (matrix):
        jumlah = 0
        for k in range (matrix):
            jumlah = jumlah + matriks1[i][k] * matriks2[k][j]
        matrikskali[i].append(jumlah)
for baris in matrikskali:
    for k in baris:
        print(k, end=' ')
    print()